blogg/posts/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Comment:
Will Django complain if we remove this file?

Reply:
Again, default Django, so left as agreed